xinference/model/llm/llm_family.py

```diff
@@ -691,7 +692,9 @@ def cache_from_csghub(
                 "model_format": llm_spec.model_format,
             },
             llm_spec.model_id,
-            endpoint="https://hub-stg.opencsg.com",
+            endpoint=os.environ.get(
+                XINFERENCE_ENV_CSG_ENDPOINT, "https://hub-stg.opencsg.com/"
```

Comment:
Shall it be `https://hub.opencsg.com/` by default?

Reply:
According to this [document](https://git-devops.opencsg.com/csg-hub-sdk/csghub-sdk
), the endpoint should be "https://hub-stg.opencsg.com/".

Reply:
But according to https://github.com/xorbitsai/inference/pull/1627#issuecomment-2172112270 they said the stg is not for production.

Reply:
I have tested and it should be using "https://hub-stg.opencsg.com/".